www/tests/check_redirects.py

```diff
@@ -22,6 +22,12 @@
     ('/site/about/ome-contributors', '/contributors'),
     ('/site/about/partners', '/commercial-partners'),
     ('/site/about/development-teams', '/teams'),
+    ('/site/about/development-teams/glencoe-software', 'https://www.glencoesoftware.com/team.html'),
+    ('/site/about/publications', '/citing-ome'),
+    ('/site/about/who-ome', '/teams'),
+    ('/site/about/what-omero/overview', '/omero'),
```

Comment:
This redirects to '/about' rather than `/omero`

Reply:
Any ideas as to why?